operations/research-and-development/product/release-process/release-scorecard-definitions.md

```diff
@@ -176,6 +177,16 @@
         </ol>
       </td>
     </tr>
+      <tr>
+      <td style="text-align:left">Average lead time per fix version</td>
+      <td style="text-align:left">
+        <p>Average number of days for all bugs with Resolution equal to Done.</p>
+        <ol>
+          <li>Project = Mattermost</li>
+          <li>Resolution = Done</li>
+        </ol>
```

Comment:
@ifoukarakis s there any other info that should be added here? (this is related to the "Average Lead Time Per Fix Version" dashboard in the Looker release metrics https://mattermost.looker.com/dashboards/410)

Reply:
The following filters are used:
- Project = Mattermost (actually project key = MM but it's practically the same)
- Issue type = Bug
- Resolution = Done
- Fix version must be defined (otherwise the issue is ignored)
- Planned release date for version must be not null ([version release dates file](https://github.com/mattermost/mattermost-data-warehouse/blob/master/transform/snowflake-dbt/seeds/version_release_dates.csv) must have been updated).
- Component must be either `Cloud` or null (on-prem). Other possible components are `IR`, `Playbooks`,  `Desktop`,  `Mobile`,  `Apps`

Reply:
Thank you!!